### ISSUE TITLE
AccountDemo not working

### ISSUE THREAD

**Issue by FunBinge** [EvilMindDevs/hms-unity-plugin#31]
Hi!

I get the following error when making a build and loading the AccountDemo scene, I am unable to sign in:

```
MissingMethodException: Constructor on type 'HuaweiMobileServices.Id.HuaweiIdAuthService' not found.
System.RuntimeType.CreateInstanceImpl (System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Object[] args, System.Globalization.CultureInfo culture, System.Object[] activationAttributes, System.Threading.StackCrawlMark& stackMark) (at <00000000000000000000000000000000>:0)
System.Activator.CreateInstance (System.Type type, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Object[] args, System.Globalization.CultureInfo culture, System.Object[] activationAttributes) (at <00000000000000000000000000000000>:0)
System.Activator.CreateInstance (System.Type type, System.Object[] args) (at <00000000000000000000000000000000>:0)
HuaweiMobileServices.Utils.AndroidJavaObjectExtensions.AsWrapper[T] (UnityEngine.AndroidJavaObject javaObject) (at <00000000000000000000000000000000>:0)
HmsPlugin.AccountManager.Start () (at <00000000000000000000000000000000>:0)
```

**Plugin Version** 2018.v1.1.1
**Unity Version:** 2018.3.14f1
**Scripting Backend:** IL2CPP

Any help appreciated.

**Comment by manelizzard** [comment]
Seems tha same error than in https://github.com/EvilMindDevs/hms-unity-plugin/issues/29

**Comment by m0skit0** [comment]
Thanks! Seems related to the SDK, creating the issue: https://github.com/m0skit0/hms-sdk-unity/issues/48

**Comment by m0skit0** [comment]
The SDK code is correct, seem more like an issue with the project itself. I can run the AccountDemo correctly in both 2018 and 2019 with IL2CPP. How are you integrating the plugin, through the repo code or the packaged releases?

**Comment by m0skit0** [comment]
Sorry, my mistake, I can reproduce the error now. This will be fixed in the SDK DLL, you can check the issue here: https://github.com/m0skit0/hms-sdk-unity/issues/49

**Comment by m0skit0** [comment]
Current fix for Unity 2019 does not work in Unity 2018 due to limitations on this one.

**Comment by m0skit0** [comment]
Fixed in commit ba7fa488e64c74b0e0bcd7e493b1e2eb956d6a77
